src/fixtures/validFile.ts

```diff
@@ -0,0 +1,7 @@
+type Config = { test?: boolean };
```

Comment:
maybe also add a fixture for a valid file that uses non-default?

Reply:
done: ea930c9